src/components/svg-canvas-graph.tsx

```diff
@@ -160,54 +165,85 @@ const SvgCanvas = () => {
                 if (isNodeSupportSnapLine(node, graph.current)) {
                     // previous move operation may have active polylines
                     // use and check if they are still valid, if not, remove and recalculate
-                    let nowPolylines = activeSnapLines;
+                    let nowSnapLines = activeSnapLines;
+                    let nowSnapPoints = activeSnapPoints;
 
                     // check if cursor left the polyline and remove it from active polylines
-                    if (nowPolylines.length !== 0) {
-                        nowPolylines = nowPolylines.filter(l => getSnapLineDistance(l, toX, toY) <= 6);
+                    if (nowSnapLines.length !== 0) {
+                        nowSnapLines = nowSnapLines.filter(l => getSnapLineDistance(l, toX, toY) <= 6);
+                    }
+
+                    if (nowSnapPoints) {
+                        if (nowSnapLines.length === 0) {
```

Comment:
What do these two conditions mean and what we should do next? Some comments should be added.